rules/do-you-know-what-happens-at-a-sprint-planning-meeting/rule.md

```diff
@@ -7,6 +7,8 @@ authors:
     url: https://ssw.com.au/people/uly
 related:
   - do-you-create-a-sprint-forecast-aka-the-functionality-that-will-be-developed-during-the-sprint
+  - do-you-conduct-an-architecture-review-after-every-sprint
+  - technical-debt
```

Comment:
should we add `just-enough-refactoring` as a related rule as well?

Reply:
Done :)